### PR TITLE
[Marvell][Nokia-M0] Fix for RIF RX_ERR Stats

### PR DESCRIPTION
**Pull request by wadoodkhan**
#### Why I did it
RIF RX_ERR counters were incrementing incorrectly.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Do not increment RIF RX_ERR, when packets are not dropped by RIF
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by Pavan-Nokia** [comment]
OC test report ID's on Kusto
M0-topo: 356d927a-15bb-4e7b-893e-0591ac98de10
Mx-topo: da9bcc8f-b9b0-43c4-ba90-74537c3da580

**Comment by Pavan-Nokia** [comment]
@wadoodkhan Couple you please change the PR to open

File: platform/marvell/sai.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@ BRANCH = 202405
 ifeq ($(CONFIGURED_ARCH),arm64)
 MRVL_SAI_VERSION = 1.14.0-3
 else ifeq ($(CONFIGURED_ARCH),armhf)
-MRVL_SAI_VERSION = 1.14.0-2
+MRVL_SAI_VERSION = 1.14.0-3
```
Comment (Blueve) [inline platform/marvell/sai.mk:7]: Has this version merged to 202405 branch?

File: device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/sai.profile [hunk 2]
```diff
@@ -3,3 +3,4 @@ hwId=et6448m
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/profile.ini
 switchProfile=/usr/share/sonic/hwsku/SAI-M0-48x1G-4x10G.xml
 createSwitchTimeout=90
+customEVlanUpdate=1
```
Comment (Blueve) [inline device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/sai.profile:6]: Can you help me understand why need the EVlan Update setting?

**Comment by Blueve** [comment]
ADO: 27364721

File: device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/sai.profile (hunk 2 above)
Comment (wadoodkhan) [reply device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/sai.profile:6]: Fix is triggered in SAI implementation based on the flag setting.

File: platform/marvell/sai.mk (hunk 1 above)
Comment (wadoodkhan) [reply platform/marvell/sai.mk:7]: This PR is targeted for 202405.

**Pull request merged**